pkgs/os-specific/linux/teck-udev-rules/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, teck-programmer }:
+
```

Comment:
```suggestion
```

Superfluous newline

Reply:
Funny... I think @SuperSandro2000 complains when they aren't there sometimes. Standardized code formatter please, right? 😆

Reply:
There were three blank lines in this file. One is sufficient.